daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector-appinsights.md

```diff
@@ -0,0 +1,88 @@
+---
+type: docs
+title: "Using OpenTelemetry Collector to collect traces to send to AppInsights"
+linkTitle: "Using the OpenTelemetry for Azure AppInsights"
+weight: 1000
+description: "How to use Dapr to push trace events to Azure Application Insights, through the OpenTelemetry Collector."
+---
+
+Dapr can integrate with [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) using the Zipkin API. This guide walks through an example to use Dapr to push trace events to Azure Application Insights, through the OpenTelemetry Collector.
+
+## Requirements
+
+A installation of Dapr on Kubernetes.
+
+## How to configure distributed tracing with Application Insights
+
+### Setup Application Insights
+
+1. First, you'll need an Azure account. See instructions [here](https://azure.microsoft.com/free/) to apply for a **free** Azure account.
+2. Follow instructions [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource) to create a new Application Insights resource.
+3. Get the Application Insights Intrumentation key from your Application Insights page.
+
+### Run OpenTelemetry Collector to push to your Application Insights instance
+
+First, find out Application Insights Instrumentation Key to be used in later steps.
```

Comment:
I would delete this sentence "his guide walks through an example to use Dapr to push trace events to Azure Application Insights, through the OpenTelemetry Collector." since it just repeats what you did in the setup

Reply:
Done